pipeline/authz/remote.go

```diff
@@ -37,9 +44,8 @@ type AuthorizerRemote struct {
 // NewAuthorizerRemote creates a new AuthorizerRemote.
 func NewAuthorizerRemote(c configuration.Provider) *AuthorizerRemote {
 	return &AuthorizerRemote{
-		c:      c,
-		client: httpx.NewResilientClientLatencyToleranceSmall(nil),
```

Comment:
Is it possible that this could cause a panic? I think we should leave the value initialized.